src/app/ipv/router.js

```diff
@@ -31,12 +29,6 @@ function getPagePath(pageId) {
 
 router.get("/usefeatureset", validateFeatureSet, renderFeatureSetPage);
 
-router.get(
```

Comment:
Is there a reason to remove this?

Reply:
`renderAttemptRecoveryPage` seems to not add anything different to what `handleJourneyPage` does for this page. It also gives the opportunity to get the `attempt-recovery` page closer to others as well by providing it with the `pageId`

Reply:
I guess the difference is it just renders the page directly without checking for session id etc?

Reply:
I see - I know we point people to it if there's an oauth error so I imagine it could be the case they don't have these and get pointed to the technical error page, losing information in the process. I will revert this 👍